### PR TITLE
create

### PR DESCRIPTION
**Pull request by chenhb179**
Signed-off-by: chb2873 <1146392873@qq.com>

### 为确保 Token Profile 的顺利通过，务必逐一确保以下检查项并补充项目资料：

**.json 及 .png 文件检查项**
- [x] logo 尺寸：120*120 px
- [x] logo 背景：透明
- [x] 代币有至少一笔成功的转账记录
- [x] 此项对于 ERC-20 项目方：合约地址有大小写（做 checksum）
- [x] .json 和 .png 文件名一致（包括大小写）

**项目资料：**

官方公告：http://www.r59iak.cn/a/xinwen/76.html
团队背景：CIB由新加坡CIB FOUNDATION LTD.发行，其创始团队由来自新加坡和中国的资深金融科技人士组成。
项目基本情况：CIB是一个“多链并存，跨链融合”的综合区块链体系，它以文化产业为切入口，用区块链全新的理念和技术，对现有文化产业的各方需求及产业发展的痛点、难点、发展瓶颈，搭建起高效、易用、分布式的区块链架构平台，服务于实体经济，满足实体经济的真实商业需求。文化产业基础链用“多链并存，跨链融合”的技术，保障落地应用场景的可行性与可靠性。从项目的开始、发展、成本、收益、回款都由智能合约自动处理，投资去向公开、透明，结合落地企业（项目）传统文化，打造 一个贯穿文化影视产业的投资、管理全流程的平台！
媒体报道：https://www.jinse.com/blockchain/533295.html
收录的交易所：99ex
转账所需 Gas Limit（对 ERC-20 项目方）：60000

---

### Please check the following to ensure a successful verification of Token Profile

**.json and .png document checklists**
- [ ] logo dimensions: 120*120 px
- [ ] logo background: Transparent
- [ ] At least one transaction on the token contract
- [ ] For ERC-20 project teams: Upper and lower capitalized alphabets in token address (For checksum purposes)
- [ ] Same document name for both .json and .png (Including capitalization)

**Project background:**

Official announcement:http://www.r59iak.cn/a/xinwen/76.html
Project team background:CIB is issued by Singapore CIB FOUNDATION LTD., And its founding team is composed of senior fintech people from Singapore and China.
Project basic information:The cultural industry basic chain (CIB) is a comprehensive blockchain system of “multi-chain coexistence and cross-chain integration”. It takes the cultural industry as the entrance, and uses the brand-new concepts and technologies of the blockchain to The needs, pain points, difficulties, and development bottlenecks of industrial development, build an efficient, easy-to-use, distributed blockchain architecture platform to serve the real economy and meet the real business needs of the real economy. The cultural industry basic chain uses the technology of "multi-chain coexistence and cross-chain fusion" to ensure the feasibility and reliability of the application scenarios. From the start, development, cost, revenue, and repayment of the project are automatically handled by smart contracts, the investment direction is open and transparent, and combined with the traditional culture of the landing enterprise (project), creating a platform that runs through the entire process of investment and management of the cultural film and television industry!
Media publications:https://www.jinse.com/blockchain/533295.html
Tradeable exchanges:99ex
Recommended Gas Limit (For ERC-20 project teams) for transaction: 60,000

### CODE REVIEW

File: erc20/0xaefb534eAEF097C41c28b58699aDb7903C602B98.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "whitepaper": "http://www.r59iak.cn/a/chanpin/fushizhubao/62.html",
+  "state": "NORMAL",
+  "published_on": "2019-01-06",
+  "initial_price":{
+        "ETH":"0.00222222 ETH",
+  },
+  "links": {
+    "medium": "https://www.jinse.com/blockchain/533295.html"
+  }
+}
```
Comment (rovca214) [inline erc20/0xaefb534eAEF097C41c28b58699aDb7903C602B98.json]: 1636 is the PR number, as in: https://github.com/consenlabs/token-profile/pull/1636 This number is the key to verify yourself as the rightful editor of the information to imToken.
The token information can only be changed after the official verification

Comment (chenhb179) [reply erc20/0xaefb534eAEF097C41c28b58699aDb7903C602B98.json]: i have updated the PR number in my website http://r59iak.cn,thank you.

**Comment by sfsy520** [comment]
非常抱歉，官网无法访问。

**Comment by sfsy520** [comment]
非常抱歉，由于该PR长时间未更新，我们先关闭这个PR
如果仍需提交，请参考教程重新提交：
https://github.com/consenlabs/token-profile#token-profile-guideline

**Pull request closed without merging**